velox/exec/Operator.cpp

```diff
@@ -286,12 +286,16 @@ OperatorStats Operator::stats(bool clear) {
   return ret;
 }
 
-void Operator::recordBlockingTime(uint64_t start) {
+void Operator::recordBlockingTime(uint64_t start, BlockingReason reason) {
   uint64_t now =
       std::chrono::duration_cast<std::chrono::microseconds>(
           std::chrono::high_resolution_clock::now().time_since_epoch())
           .count();
-  stats_.wlock()->blockedWallNanos += (now - start) * 1000;
+  auto wallNanos = stats_.wlock()->blockedWallNanos += (now - start) * 1000;
```

Comment:
```
const auto wallNanos = (now - start) * 1000;
stats_.wlock()->blockedWallNanos += wallNanos;
const auto statName = fmt::format("{}TimeNs", blockingReasonToString(reason));
```

Reply:
I'll split the assignment.

On the other hand, we might have to use the blocked{}WallNanos, as I see the UI do the time formatting when seeing "Nanos"